docroot/profiles/custom/cgov_site/modules/custom/cgov_core/tests/src/Kernel/FieldStorage/CGovFieldStorageTest.php

```diff
@@ -142,6 +142,7 @@ public function testFields() {
     // Test the field persistance when it has been removed from all nodes.
     foreach ($this->fieldsToTest as $fieldToTest) {
       // Remove from our content type.
+      /** @var \Drupal\Core\Entity\EntityInterface $field */
```

Comment:
I have tried removing this and checking $field as instance of a EntityInterface but not sure why the error not gone away.

@bryanpizzillo